src/scrape.ts

```diff
@@ -125,7 +125,13 @@ const main = async () => {
     console.log(`Loaded ${webhooks.length} webhooks for version ${version}`);
 
     for (const webhook of webhooks) {
-      webhook.response = JSON.parse(webhook.response);
+      if (webhook.response === "") {
+        console.warn(`${webhook.name} has an empty response`);
+        webhook.response = {};
+      } else {
+        webhook.response = JSON.parse(webhook.response);
+      }
```

Comment:
Woops just noticed I should update this to increment the warning count, but this is the quick fix.